github-runner-manager/tests/integration/helper.py

```diff
@@ -28,12 +28,16 @@ def get_app_log(app: subprocess.Popen) -> str:
     return app.stderr.read().decode("utf-8")
 
 
-def start_app(config_file: Path, extra_args: Sequence[str]) -> subprocess.Popen:
+def start_app(
+    config_file: Path, extra_args: Sequence[str], wait_flask_start: bool = False
+) -> subprocess.Popen:
     """Start the CLI application.
 
     Args:
         config_file: The Path to the configuration file.
         extra_args: Any extra args to the CLI application.
+        wait_flask_start: Wait for flask to start before returning. Logs prior to the flask server
+            started will be removed.
```

Comment:
How/why will the logs be removed prior to the start?

Reply:
The waiting for flask to start involves reading the logs until it prints a log on flask being ready.

Reply:
I will change it to checking on a health endpoint.